packages/server/src/api/controllers/row/ExternalRequest.ts

```diff
@@ -572,11 +582,18 @@ export class ExternalRequest {
       if (!linkTable || !linkPrimary) {
         return
       }
+
+      const linkSecondary =
+        linkTable?.primary &&
+        linkTable?.primary?.length > 1 &&
+        linkTable?.primary[1]
+
       const rows = related[key]?.rows || []
       const found = rows.find(
```

Comment:
I'm struggling to understand this section of the code. Can the variable name be improved or some of this moved into a function? I'm not sure what has been "found".